jenkins/Jenkinsfile-blossom.premerge-databricks

```diff
@@ -1,6 +1,6 @@
 #!/usr/local/env groovy
 /*
- * Copyright (c) 2023, NVIDIA CORPORATION.
+ * Copyright (c) 2023-2024, NVIDIA CORPORATION.
```

Comment:
Check copyrights in every file. Since you haven't used the pre-commit, the easiest is probably to run 

```bash
./scripts/auto-copyrighter.sh $(git diff --name-only origin/branch-24.04..HEAD)
```

Reply:
Thanks! My pre-commit is broken somehow and I assumed it ran.